cmake/patches/composable_kernel/Fix_Clang_Build.patch

```diff
@@ -1,5 +1,5 @@
 diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
Shall we delay this PR so we can also include the `IsSupported` fix from ck?

Reply:
Yes, we must delay. Because the new version of CK breack our CI now. we need to complete the todo list in description, especially the 1 and 2.